setup.py

```diff
@@ -12,12 +12,31 @@
 needs_pytest = {'pytest', 'test', 'ptr'}.intersection(sys.argv)
 pytest_runner = ['pytest-runner'] if needs_pytest else []
 
+# Check Python version
+if sys.version_info[:2] < (3, 6):
+    raise RuntimeError('Python version >=3.6 required.')
+
+# Add release
+RELEASE = "0.0.0-dev0"
```

Comment:
I think this is probably not required -- MDAnalysis core uses it to

* set up the version in the final `setup()` call
* specify which version of the tests is required
* check if it's a release version (no "dev") or development version.

You probably don't need any of those unless you plan to manage concurrent released and development versions the way MDAnalysis does, with its own docs and C/++ extensions. As you are using ReadTheDocs for documentation currently, RTD manages your doc versioning for you.